### PR TITLE
Remove statistics information and change firewalld description in TiSpark Overview

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

- Fix format: white space, spark-sql code format, link of http://spark-master-hostname:8080
- Remove note and config in statistics information
- Change firewalld description to a note

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tispark/pull/2300

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tispark-overview.md [hunk 1]
```diff
@@ -127,13 +127,9 @@ The `spark.tispark.pd.addresses` configuration allows you to put in multiple PD
...
-The default setup of `firewalld` on CentOS blocks the traffic needed by TiSpark. Therefore, disable `firewalld` to ensure normal running of TiSpark.
-
-```bash
-sudo systemctl stop firewalld.service
-sudo systemctl disable firewalld.service
-sudo systemctl mask --now firewalld.service
-```
+> **Note:**
+>
+> If TiSpark could not communicate properly, please check your firewall configuration. You can adjust the firewall rules or disable it on your need.
```
Comment (xuanyu66) [inline tispark-overview.md:132]:
```suggestion
> If TiSpark could not communicate properly on CentOS, please check your firewall configuration. You can adjust the firewall rules or disable it on your need.
```

**Review by xuanyu66 — CHANGES_REQUESTED** [review]

File: tispark-overview.md (hunk 1 above)
Comment (Oreoxmt) [reply tispark-overview.md:132]: This is not a OS specific problem, so this change won't be accepted.

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**